base/applications/rapps/installed.cpp

```diff
@@ -88,7 +88,7 @@ void CInstalledApplicationInfo::EnsureDetailsLoaded()
         if (GetApplicationRegDword(L"WindowsInstaller", &dwWindowsInstaller) && dwWindowsInstaller)
         {
             // MSI has the same info in Uninstall / modify, so manually build it
-            szUninstallString.Format(L"msiexec /x%s", m_szKeyName.GetString());
+            szUninstallString.Format(L"msiexec /x %s", m_szKeyName.GetString());
```

Comment:
There is a special case in our (and wine's) code to handle exactly this case,
were any negative effects observed from the two options being concatenated?
https://git.reactos.org/?p=reactos.git;a=blob;f=base/system/msiexec/msiexec.c;hb=95e5f07084088129e3c121568ea06c7f4d0a8d41#l257

Reply:
Sorry I missed this specific case being properly managed by ReactOS/Wine. 

Do you then consider this change is un-nnecesasry and to be removed from this PR (and associated ticket closed as Won't do) ? Or is the change "cleaner" (but only cosmetic) ?

Reply:
This looks like a change for the sake of changing it, hence my original question in [CORE-18774](https://jira.reactos.org/browse/CORE-18774):
> And this is a problem because?

Reply:
This is not a problem and I didn't knew ReactOS specifically handle this case. Not sure Windows does but likely not a sufficient reason for change. 

In this case I propose to trim this PR to the Hyperlink fix only

@rexxDigital : ok for you ?

Reply:
Yeah thats ok with me

Reply:
This has worked fine for years as far as I know, and I have used rapps both on windows and on ros to test stuff.